css/style.css

```diff
@@ -464,10 +496,270 @@ div .taggs {
+    font-family: 'Poppins', sans-serif;
+    background: #e5e5e5;
+  }
+
+  #main-page {
+    left: 0;
+    height: 800px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: #fff;
+    background-image: url(/images/desktop_main/Header.svg);
+    background-size: 97%;
+    background-repeat: no-repeat;
+    background-position: 20px 80px;
+    margin-bottom: 4rem;
+    padding: 10px 24px;
+  }
+
+  #navbar {
+    overflow: hidden;
+    background-color: #fff;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    margin-left: 8px;
+  }
+
+  .my-logo {
+    margin: 16px 0 0 100px;
+  }
+
+  .navbar-menu {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    gap: 1rem;
+    margin-right: 10%;
+  }
+
+  .navbar-menu a {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .headline {
+    padding: 350px 260px 255px;
+    margin-top: 20px;
+  }
+
+  /* ////// Projects /////// */
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem 0;
+  }
+
+  .work-page-left {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    align-content: center;
+    width: 80%;
+    background: #fff;
+    border: none;
+    border-radius: 24px;
+    padding: 2rem;
+  }
+
+  .work-page-right {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    background: #fff;
+    border: none;
+    border-radius: 24px;
+    padding: 2rem;
+  }
+
+  .card1-content,
+  .card3-content {
+    margin: 30px 20px 0 0;
+  }
+
+  .card2-content {
+    grid-row: 1/-2;
+    margin: 30px 20px 0 0;
+    padding-left: 15px;
+  }
+
+  .work-title {
+    width: 100%;
+  }
+
+  .card1-image,
+  .card2-image,
+  .card3-image {
+    transition: transform 0.3s;
+  }
+
+  .card1-image:hover,
+  .card2-image:hover,
+  .card3-image:hover {
+    transform: scale(1.07);
+  }
+
+  .clients {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    padding: 0 15px;
+    margin: 1px;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  div .taggs {
+    float: right;
+    width: 40px;
+    height: 20px;
+    padding-bottom: 20px;
+  }
+
+  .tag1 {
+    width: 50px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    margin-bottom: 20px;
+    color: #396df2;
+  }
+
+  .tag2 {
+    width: 40px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    color: #396df2;
+  }
+
+  .tag3 {
+    width: 80px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    color: #396df2;
+  }
+
+  /* //////// About-myself-desktop//////// */
+
+  .about-myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
- [ ] Kindly consider writing this line here `grid-template-columns: 40% 60%;`. This will make your design look much similar to the Figma design. But in this case, you will need to lower the padding on the left and right sides of the container from both sides to make it look better.